### ISSUE TITLE
Table names and improved documentation

### ISSUE THREAD

**Issue by javiertury** [rmp135/sql-ts#49]
I think that exporting an enum for table names is useful. However it's very difficult to make everyone happy because the format varies too much depending on your software configuration.

For instance, let's say that the configuration is extended to provide the following extra properties for a table.

- `keyedName`: human friendly name for the table (possibly camelCase transformed, possibly including the schema to avoid table name collisions)
- `exportedName`: table name used for queries (possibly camelCase transformed)
- `exportedSchema`: schema name used for queries (possibly camelCase transformed)

Knex requires schema and table_name to be exported separately and possibly with camelCase transformations.

```
export const TableNames = {
  {{#each grouped as |group key|}}
  {{#each tables as |table|}}
  {{keyedName}}: {
    schema: "{{schema}}",
    name: "{{name}}"
  }{{#if @last}}{{else}},{{/if}}
  {{/each}}
  {{/each}}
}
```

However slonik and plain sql queries use the "schema.table_name" format.

```
export enum TableNames {
  {{#each grouped as |group key|}}
  {{#each tables as |table|}}
  {{keyedName}} = "{{schema}}.{{name}}"{{#if @last}}{{else}},{{/if}}
  {{/each}}
  {{/each}}
}
```

I also think that the documentation needs a simpler example of a handlebars template for those of us that are not familiar with the format. It believe it can help users to better visualize a solution to their problem.

TLDR: I would like to contribute a useful and illustrative example for a handlebars template in #50
